src/viur/core/skeleton.py

```diff
@@ -873,185 +873,179 @@ def toDB(cls, skelValues: SkeletonInstance, update_relations: bool = True, **kwa
 
             :returns: The datastore key of the entity.
         """
-        assert skelValues.renderPreparation is None, "Cannot modify values while rendering"
+        assert skel.renderPreparation is None, "Cannot modify values while rendering"
         # fixme: Remove in viur-core >= 4
         if "clearUpdateTag" in kwargs:
             msg = "clearUpdateTag was replaced by update_relations"
             warnings.warn(msg, DeprecationWarning, stacklevel=3)
             logging.warning(msg, stacklevel=3)
             update_relations = not kwargs["clearUpdateTag"]
 
-        def txnUpdate(dbKey, mergeFrom):
-            skel = mergeFrom.skeletonCls()
-
-            blobList = set()
-            changeList = []
+        def txnUpdate(write_skel):
+            db_key = write_skel["key"]
+            skel = write_skel.skeletonCls()
 
+            blob_list = set()
+            change_list = []
+            old_copy = {}
             # Load the current values from Datastore or create a new, empty db.Entity
-            if not dbKey:
+            is_add = not bool(db_key)
+            if is_add:
```

Comment:
Please use modern syntax!
```suggestion
            if is_add := not bool(db_key):
```